org-code-javabuilder/validation/src/main/java/org/code/validation/support/UserTestResultSignalKey.java

```diff
@@ -0,0 +1,6 @@
+package org.code.validation.support;
+
+public enum UserTestResultSignalKey {
+  TEST_STATUS,
```

Comment:
nit: should this be TEST_RESULT? doesn't really matter, but just thinking of a case where we would need to send status messages as a test is happening separately from the test results

Reply:
I'm going to keep it as is for 2 reasons: the java `TestExecutionResult` uses "status" to refer to success/failure/aborted, and I already merged the Java Lab change :)